AltStore/AppDelegate.swift

```diff
@@ -33,10 +33,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
     
-    @available(iOS 14, *)
```

Comment:
And you're removing the selective availability, because we can now assume people are only building for iOS 14 and above.